src/main/java/cam72cam/immersiverailroading/render/item/LargeWrenchModel.java

```diff
@@ -0,0 +1,137 @@
+import net.minecraft.client.renderer.block.model.ItemCameraTransforms.TransformType;
+import net.minecraft.client.renderer.block.model.ItemOverride;
+import net.minecraft.client.renderer.texture.TextureAtlasSprite;
+import net.minecraft.entity.EntityLivingBase;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.EnumFacing;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+import net.minecraftforge.client.ForgeHooksClient;
+import util.Matrix4;
+
+public class LargeWrenchModel implements IBakedModel {
+	private static OBJRender RENDERER;
+	
+	private OBJRender renderer;
+	
+	public LargeWrenchModel() {
+		if (RENDERER == null) {
+			try {
+				RENDERER = new OBJRender(new OBJModel(new ResourceLocation("immersiverailroading:models/item/wrench/wrench.obj"), -0.5f));
+			} catch (Exception e) {
+				e.printStackTrace();
+			}
+		}
+		
+		this.renderer = RENDERER;
+	}
+
+	@Override
+	public List<BakedQuad> getQuads(IBlockState state, EnumFacing side, long rand) {
+		if (this.renderer != null) {
+			GLBoolTracker cull = new GLBoolTracker(GL11.GL_CULL_FACE, false);
+			GLBoolTracker lighting = new GLBoolTracker(GL11.GL_LIGHTING, false);
+
+			GL11.glPushMatrix();
+			this.renderer.bindTexture();
+			this.renderer.draw();
+			this.renderer.restoreTexture();
+			GL11.glPopMatrix();
+
+			lighting.restore();
+			cull.restore();
+			
+			this.renderer = null;
+		}
+		
+		return new ArrayList<BakedQuad>();
+	}
+
+	@Override
+	public boolean isAmbientOcclusion() {
+		return true;
+	}
+
+	@Override
+	public boolean isGui3d() {
+		return true;
+	}
+
+	@Override
+	public boolean isBuiltInRenderer() {
+		return false;
+	}
+
+	@Override
+	public TextureAtlasSprite getParticleTexture() {
+		return null;
+	}
+	
+	public class ItemOverrideListHack extends ItemOverrideList {
+		public ItemOverrideListHack() {
+			super(new ArrayList<ItemOverride>());
+		}
+
+		@Override
+		public IBakedModel handleItemState(IBakedModel originalModel, ItemStack stack, @Nullable World world, @Nullable EntityLivingBase entity) {
+			return new LargeWrenchModel();
+		}
+	}
+
+	@Override
+	public ItemOverrideList getOverrides() {
+		return new ItemOverrideListHack();
+	}
+
+	@Override
+	public Pair<? extends IBakedModel, Matrix4f> handlePerspective(TransformType cameraTransformType) {
+		Pair<? extends IBakedModel, Matrix4f> defaultVal = ForgeHooksClient.handlePerspective(this, cameraTransformType);
+		switch (cameraTransformType) {
+		case THIRD_PERSON_LEFT_HAND:
+		case THIRD_PERSON_RIGHT_HAND:
+			return Pair.of(defaultVal.getLeft(),
+					new Matrix4().rotate(Math.toRadians(90), 0, 1, 0).translate(1.5, 0.375, 1.5).scale(3, 3, 3).toMatrix4f());
+		case FIRST_PERSON_LEFT_HAND:
+		case FIRST_PERSON_RIGHT_HAND:
+			return Pair.of(defaultVal.getLeft(), new Matrix4().rotate(Math.toRadians(90), 0, 1, 0).rotate(Math.toRadians(-45), 0, 0, 1).translate(21, 1, 23.2).scale(35, 35, 35).toMatrix4f());
```

Comment:
This is different for LEFT/RIGHT hand.  Please test this in game.
![2019-06-18_07 38 36](https://user-images.githubusercontent.com/892136/59679378-a8893200-919c-11e9-8d29-00b8bd9a1e06.png)